build/includes/sdk.mk

```diff
@@ -34,8 +34,7 @@ SDK_FOLDER ?= go
 COMMAND ?= gen
 SDK_IMAGE_TAG=$(build_sdk_prefix)$(SDK_FOLDER):$(build_sdk_version)
 DEFAULT_CONFORMANCE_TESTS = ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch,reserve
-ALPHA_CONFORMANCE_TESTS = getplayercapacity,setplayercapacity
-
+ALPHA_CONFORMANCE_TESTS = getplayercapacity,setplayercapacity,playerconnect,playerdisconnect,getplayercount,isplayerconnected,getconnectedplayers
```

Comment:
it looks like these don't have to be in the same order as the code. do we just check that they were all called?

Reply:
You are right, it doesn't matter the order they are called in - it checks that SDK methods are called with the expected arguments.